src/components/SelectMonths.js

```diff
@@ -0,0 +1,17 @@
+export default function SelectMonths({ monthNames, months, callBack }) {
+  function handleOnClickNewMonth(event) {
+    const month = event.target.value;
+    callBack(month);
+  }
+
+  let renderedButtons = [];
+  for (let i = 0; i < months.length; i++) {
+    const monthButton = (
+      <button onClick={handleOnClickNewMonth} key={i} value={months[i]}>
+        {monthNames[months[i]]}
+      </button>
+    );
+    renderedButtons.push(monthButton);
+  }
```

Comment:
Try to use modern JavaScript to write a loop. It is not wrong though, but a map is way more readable. And you're saving a lot of lines. ;-)
```suggestion
  let renderedButtons = months.map((month, index) => (
      <button onClick={handleOnClickNewMonth} key={index} value={month}>
        {monthNames[month]}
      </button>))
```